### ISSUE TITLE
Is the validation loss computed in the mmsegmentation 1.2.2?  How do you know if the model in training has converged

### ISSUE THREAD

**Issue by eighteenlin** [open-mmlab/mmsegmentation#3679]
Hello,
I am using mmseg version 1.2.2.

How can you tell if the model in training has converged? 

I sincerely wonder how to get the validation loss and the accuracy score in the training step.

I already know how to get the validation set loss in mmsegmentation old version? How do I get the new version?

Can we use loss as an evaluation metric in the validation set and combine it with the loss in the training set to determine model convergence?

Thanks for any help.
